pyproject.toml

```diff
@@ -45,14 +45,14 @@ nowarn = "test -W default {args}"
 
 [tool.hatch.envs.cov]
 features = ["test"]
-dependencies = ["coverage", "pytest-cov"]
+dependencies = ["coverage", "pytest-cov", "argcomplete"]
```

Comment:
"argcomplete" is already handled by the "test" feature, here and in the typing env.